index.html

```diff
@@ -17,25 +17,40 @@
 
     <header class="menu">
         <nav>
-            <img class="hamb" src="./images/ic_menu.svg" alt="hamburguer">
+                <img class="hamb" src="./images/ic_menu.svg" alt="hamburguer">
+            <ul class="desk-bar">
+                <li><a class="top-a" href="#greetings">Hello</a></li>
+                <li><a class="top-a" href="#Projects">Portfolio</a></li>
+                <li><a class="top-a" href="#mainabout">About</a></li>
+                <li><a class="top-a" href="#initial2">Contact</a></li>
+            </ul>
         </nav>
     </header>
 
     <main>
         <section id="greetings">
-            <h1>Juan Pablo Guerrero Rios</h1>
-            <p class="firstp"> Hello! I am a software developer! I can help you build a product, feature or website.
+            
+            <div id="initial">
+                <h1>Juan Pablo Guerrero Rios</h1>
+                <p class="firstp"> Hello! I am a software developer! I can help you build a product, feature or website.
```

Comment:
- [OPTIONAL ] Your paragraph `firstp` is moving all the way to the right. 
 
![Screenshot from 2022-11-04 17-40-43](https://user-images.githubusercontent.com/89228743/200043041-fd1b2951-788e-43a4-9a1a-3ed921618239.png)

 Please I would highly recommend you fix it to make your project look more like the template.  What you can do is in your media query give it a                        
 `padding-left: 160px;`. 👌